python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -134,12 +145,17 @@ class GrapheneAssetNode(graphene.ObjectType):
         partitions=graphene.List(graphene.String),
     )
     materializationCountByPartition = non_null_list(GrapheneMaterializationCount)
+    materializationCountByDimensionPartition = graphene.Field(
```

Comment:
preference would be to just fold this into the above graphql field with an optional fixed dimension value arg, and then do an assert based on the partitions def that N-1 of the dimension values are populated.

Reply:
I was able to merge the two resolvers into a `partitionMaterializationCount` resolver! There's also an internal check that ensures that each dimension value is populated for a given event.